pytorch_toolkit/instance_segmentation_2/model_templates/coco-instance-segmentation/instance-segmentation-0002/modules.yaml

```diff
@@ -0,0 +1,5 @@
+arg_parser: DefaultArgParser
+arg_converter: InstanceSegmentationArgsConverter
```

Comment:
Having mmdetection counterparts is not enough?

Reply:
`ArgsConverter` for instance segmentation has a couple of differences related to dataset parameters and additional head - `mask_head`. I can use `MMDetectionArgsConverter` but with some fixes like:
```python
dataset = 'dataset.' if 'dataset' in cfg.data.train.keys() else ''
    update_config_map = {
        'train_ann_files': f'data.train.{dataset}ann_file',
        'train_data_roots': f'data.train.{dataset}img_prefix'}
```
```python
if 'mask_head' in cfg.model.roi_head.keys():
    update_config['model.roi_head.mask_head.num_classes'] = num_clusses
```
But I am not sure that this way is better.

Reply:
I would warn from creating omnipotent solutions... In this particular case the copy-paste is lesser evil - no potential troubles for others.

Reply:
@Ilya-Krylov, what do you think?

Reply:
I don't think that it is omnipotent, I think that instance segmentation models and object detection models from mmdetection can be handled by one 'solution' though I do not insist.

Probably one more opinion, @druzhkov-paul ?

Reply:
Fixed. I applied `Repeat dataset` and added only `mask_head` section.